examples/complete/main.tf

```diff
@@ -88,7 +88,7 @@ module "ecs_web_app" {
   authentication_oidc_token_endpoint            = var.authentication_oidc_token_endpoint
   authentication_oidc_user_info_endpoint        = var.authentication_oidc_user_info_endpoint
 
-  # ECS
+  // ECS
```

Comment:
Why are we reverting to the slash comments?

Reply:
Just generally reverting to TF 0.12 support by reverting some commits. No special reason.

Reply:
`context.tf` has the // anyway, so it's not going to pass lint. Will fix when we update `context.tf`